requirements/base.txt

```diff
@@ -4,6 +4,6 @@
 #
 #    make upgrade
 #
-django==2.2.14            # via -c requirements/constraints.txt, -r requirements/base.in
-pytz==2020.1              # via django
-sqlparse==0.3.1           # via django
+django==2.2.17            # via -c requirements/constraints.txt, -r requirements/base.in
```

Comment:
Looks like we still need a `make upgrade` job for this repo.

Reply:
Ran `make upgrade` again to fetch the new versions.